### ISSUE TITLE
[Bug]: The required namespace "day8.re-frame-10x.preload.react-18" is not available.

### ISSUE THREAD

**Issue by saburto** [day8/re-frame-10x#407]
### What happened?

Hi people!

I followed the recomandation of using the preload `day8.re-frame-10x.preload.react-18` for react-18 projects according to the README, but sadly I can't compile because I have the following error:

```
The required namespace "day8.re-frame-10x.preload.react-18" is not available.
```

I am using the release `1.8.1` but it looks like it doesn't contain the latest changes, even the latest snapshot has not the latest changes you can see in clojars https://clojars.org/day8.re-frame/re-frame-10x/versions/1.8.3-SNAPSHOT is based on the git tree https://github.com/day8/re-frame-10x/tree/ce399742256106cadab39787efb5e122fd234248

Any plans to release the latest changes?

Thanks!

### Factory Reset

- [ ] yes

### 10x Version

1.8.1

### Reagent Version

1.2.0

### React Version

1.8

### re-frame Version

1.4.0

### Relevant console output

```shell
The required namespace "day8.re-frame-10x.preload.react-18" is not available.
```

**Comment by kimo-k** [comment]
Let me see if I can cut a release today, otherwise maybe another snapshot. Hadn't thought of this before - it is kind of problematic that the readme is in sync with master and not the latest release.

**Comment by kimo-k** [comment]
Hey, you can try out https://clojars.org/day8.re-frame/re-frame-10x/versions/1.9.0-SNAPSHOT for now. Watch out for misbehaving input fields and right-click menus. Still discovering the react 18 quirks.

**Comment by kimo-k** [comment]
Fixed in 1.9.0

**Comment by saburto** [comment]
Thanks!
